inc/plugins/namespace.php

```diff
@@ -0,0 +1,99 @@
+<?php
+/**
+ * Sets DID-less plugin slug as active.
+ *
+ * @package FAIR
+ */
+
+namespace FAIR\Plugins;
+
+use function FAIR\Packages\get_did_hash;
+use function FAIR\Updater\get_packages;
+
+/**
+ * Bootstrap.
+ *
+ * @return void
+ */
+function bootstrap() {
+	add_action( 'load-plugins.php', __NAMESPACE__ . '\\load_filters' );
+}
+
+/**
+ * Load filters.
+ *
+ * @return void
+ */
+function load_filters() {
+	add_filter( 'option_active_plugins', __NAMESPACE__ . '\\set_as_active' );
```

Comment:
I think this active plugins hook should still be in the bootstrap since that option is accessed in many more places, whereas the admin notice is only on `plugins.php`